components/shared/PrivacyPolicy.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+import { Trans, useTranslation } from 'next-i18next';
+
+export const PrivacyPolicy = () => {
+  const { t } = useTranslation('components');
+
+  return (
+    <Box textAlign="center">
+      <Typography color="textSecondary" variant="body2">
+        <Trans i18nKey="links.privacyPolicy" t={t}>
```

Comment:
You don't need these Trans components anymore since these are all links with a single word inside

Reply:
Even if it's a single word, if you wanted to eventually support i18n then you need it